### ISSUE TITLE
ExtensionInfoEvents.ExtensionInfoEventsBody.hints outdated

### ISSUE THREAD

**Issue by u9520107** [ringcentral/ringcentral-extensible#4]
There are some issues with this definition:
1. Link to the current spec in README.md is outdated
2. According to the spec:
```
hints:
        type: array
        description: Returned for 'Update' event type only
        items:
          type: string
          enum:
            - AccountSettings
            - AccountStatus
            - AnsweringRules
            - CompanyNumbers
            - DialingPlan
            - ExtensionInfo
            - Features
            - Limits
            - Permissions
            - ProfileImage
            - VideoConfiguration
```
However, the in the current release (0.8.6 of ringcentral-extensible):
```
hints?: ('LimitsFeatures' | 'AccountSettings' | 'CompanyNumbers' | 'AccountStatus' | 'DialingPlan' | 'Permissions' | 'ProfileImage' | 'ExtensionInfo' | 'VideoConfiguration')[];
```
Limits and Features are combined in the current release, but are separated in the spec.

**Comment by tylerlong** [comment]
The spec link in readme file has been fixed.  Yesterday I just updated the C# SDK to use latest spec. And this project will share the same spec file as the C# SDK.

**Comment by tylerlong** [comment]
This is copied from latest spec:

```yaml
      hints:
        type: array
        description: Returned for 'Update' event type only
        items:
          type: string
          enum:
            - AccountSettings
            - AccountStatus
            - AnsweringRules
            - CompanyNumbers
            - DialingPlan
            - ExtensionInfo
            - Features
            - Limits
            - Permissions
            - ProfileImage
            - VideoConfiguration
```

**Comment by tylerlong** [comment]
Latest code generated:

```ts
  /**
   * Returned for 'Update' event type only
   */
  hints?: (
    | 'AccountSettings'
    | 'AccountStatus'
    | 'AnsweringRules'
    | 'CompanyNumbers'
    | 'DialingPlan'
    | 'ExtensionInfo'
    | 'Features'
    | 'Limits'
    | 'Permissions'
    | 'ProfileImage'
    | 'VideoConfiguration'
  )[];
```

**Comment by tylerlong** [comment]
Released 0.9.0
